autonomy/chain/service.py

```diff
@@ -89,11 +92,63 @@ def get_service_info(
     )
 
 
+def wait_for_success_event(
```

Comment:
In the same manner we will also wait for the success event for various service operations